R/merge.R

```diff
@@ -151,8 +152,8 @@ setGeneric("mergeCols",
     FUN <- function(mat, f = f, ...){
         temp <- scuttle::summarizeAssayByGroup(mat, 
                                                ids = f, 
-                                               subset_row = NULL, 
-                                               subset_col = NULL, 
+                                               subset.row = NULL, 
+                                               subset.col = NULL, 
                                                statistics = "sum",
                                                ...)
```

Comment:
I think this `summarizeAssayByGroup` does this agglomeration for only column-wise, so there shoulld be transpose step?

mat --> t(mat)